crates/witness-generator/src/eest_generator.rs

```diff
@@ -170,7 +166,7 @@ impl WitnessGenerator for ExecSpecTestBlocksAndWitnesses {
                 let chain_config = chain_spec.genesis.config;
                 Ok(BlockAndWitness {
                     name: name.to_string(),
-                    block_and_witness: run_case(case)?
+                    block_and_witness: BlockchainTestCase::run_single_case(name, case)?
```

Comment:
We switch to `BlockchainTestCase::run_single_case`. This method (https://github.com/eth-act/zkevm-benchmark-workload/pull/150) semantics is that it only returns an error if the test fails. 

Note that if the test has invalid blocks and Reth failed (as expected), that means the test run is successful.

This means that from our perspective here, we always expect all fixtures to succeed (we don't care if they have valid or invalid blocks). The returned witness is always everything needed to execute the block (i.e., full or partial).